sources/packages/web/src/components/common/OfferingForm.vue

```diff
@@ -2,53 +2,105 @@
   <formio-container
     formName="educationProgramOffering"
     :formData="formData"
-    @submitted="saveOffering"
+    @loaded="$emit('loaded', $event)"
+    @changed="$emit('changed', $event, $event)"
```

Comment:
why `$event, $event`?

Reply:
This syntax worked for me in the same way that the below one. I can change to the below one maybe because it is easier to understand.
```ts
@changed="(form, event) => $emit('changed', form, event)"
```